tests/unit/net_test.ts

```diff
@@ -1204,33 +1203,22 @@ Deno.test({
       p1 = listener1.accept().then((conn) => {
         conn.close();
```

Comment:
This was assuming that connections are always round-robin'd, but that's not a guarantee.